src/components/AvatarList.stories.js

```diff
@@ -50,7 +50,7 @@ export const SmallSize = Basic.bind();
 SmallSize.args = { users, userCount: 100, size: 'small' };
 
 export const Loading = Basic.bind();
-Loading.args = { users: [], isLoading: true };
+Loading.args = { users, userCount: 100, isLoading: true };
```

Comment:
```suggestion
Loading.args = { users: undefined, userCount: undefined, isLoading: true };
```

Just took a look at this and I think the `users` and `userCount` are meant to be left undefined for the loading state since in theory we wouldn't know these values during loading.

Reply:
yes, makes sense. but then, it should be only 
```suggestion
Loading.args = { users: undefined, isLoading: true };
```
as the `userCount` prop is optional!

Reply:
Nice one. Sounds good!